src/ml/neural_net/tf_compute_context.cpp

```diff
@@ -1,15 +1,14 @@
-/* Copyright © 2018 Apple Inc. All rights reserved.
+/* Copyright © 2019 Apple Inc. All rights reserved.
  *
  * Use of this source code is governed by a BSD-3-clause license that can
  * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
 */
+#include <ml/neural_net/tf_compute_context.hpp>
 
 #include <iostream>
-#include <vector>
-
-#include <ml/neural_net/tf_compute_context.hpp>
 #include <pybind11/pybind11.h>
```

Comment:
Let's follow https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes and move the pybind includes after the system includes